core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -1132,6 +1133,16 @@ public void format() throws IOException {
     }
   }
 
+  @Override
+  public String toString() {
+    return MoreObjects.toStringHelper(this)
+        .add("Journal Path", mConf.getPath())
+        .add("State", mPrimarySelector.getState())
+        .add("ClientId", mRawClientId)
```

Comment:
this is a random Id right? Did you find it helpful?

Reply:
Yeah, maybe it can be removed.